### PR TITLE
Add the 'cloud launch blocks local launch' bug to known issues

### PR DESCRIPTION
**Pull request by BenNaccarato**
#### Description / Documentation
Add a brief description of the known issue https://improbableio.atlassian.net/browse/UNR-1006 to the readme

#### Primary reviewers
@girayimprobable @NotRobSheridan

### CODE REVIEW

File: SpatialGDK/Documentation/known-issues.md [hunk 1]
```diff
@@ -18,3 +18,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Sometimes the player is unable to move when spawning | 2018-10-30 | [UNR-691](https://improbableio.atlassian.net/browse/UNR-691) | Reconnect| 
 | Seamless Travel is not supported | 2019-01-22 | [UNR-897](https://improbableio.atlassian.net/browse/UNR-897) | Disable Seamless travel |  
 | Gameplay Ability System is not fully supported | 2019-02-01 | | Use the workarounds detailed on the [Gameplay Ability System]({{urlRoot}}/content/ability-system) reference page. |  
+| Launching the editor whilst uploading or deploying to cloud blocks the in-editor 'Start' button, and can be stopped from the in-editor 'Stop' | 2019-02-18 | [UNR-1006](https://improbableio.atlassian.net/browse/UNR-1006) | Wait for the cloud launch to finish, or start the cloud launch whilst the editor is already active. |
```
Comment (NotRobSheridan) [inline SpatialGDK/Documentation/known-issues.md]:
```suggestion
| Opening the Editor whilst uploading or launching a cloud deployment disables the GDK Toolbar **Start** button. The deployment can still be stopped by selecting **Stop** on the GDK Toolbar. | 2019-02-18 | [UNR-1006](https://improbableio.atlassian.net/browse/UNR-1006) | Wait for the cloud launch to finish, or launch the cloud deployment whilst the Editor is already open. |  
```
Does the **Start** button stay disabled after pressing **Stop**?

Comment (BenNaccarato) [reply SpatialGDK/Documentation/known-issues.md]: The start button is enabled after you press stop, but that does kill the cloud launch, which isn't what it should do 😅

Comment (NotRobSheridan) [reply SpatialGDK/Documentation/known-issues.md]: Curiouser and curiouser. 

Maybe: 

```suggestion
| Opening the Editor while you are uploading or launching a cloud deployment disables the GDK Toolbar **Start** button, and Selecting **Stop** on the GDK Toolbar stops the running cloud deployment. | 2019-02-18 | [UNR-1006](https://improbableio.atlassian.net/browse/UNR-1006) | Wait for the cloud launch to finish, or launch the cloud deployment whilst the Editor is already open. |```

Comment (BenNaccarato) [reply SpatialGDK/Documentation/known-issues.md]: Yeah, that sounds good. What is the suggested ``` at the end for?

**Pull request merged**